modules/core/src/test/java/org/apache/ignite/internal/metric/CacheMetricsAddRemoveTest.java

```diff
@@ -152,12 +200,25 @@ private void createCache(@Nullable String dataRegionName, @Nullable String cache
         if (cacheName != null)
             ccfg.setName(cacheName);
 
+        if (groupName != null)
+            ccfg.setGroupName(groupName);
+
         if (nearEnabled)
             ccfg.setNearConfiguration(new NearCacheConfiguration());
 
         grid("client").createCache(ccfg);
 
         awaitPartitionMapExchange();
+
```

Comment:
Can we just pass nearEnabled into cacheMetricsRegistryName instead of if statement?

Reply:
No, we can't. We should configure cache and near cache metrics.